deployment/upgradeLive.sh

```diff
@@ -21,6 +20,7 @@ if [ ! -d "$1" ] ; then
         unlink /opt/kaltura/liveController/latest
```

Comment:
maybe, as you suggested, unlink "latest" as a first step when running the script